### ISSUE TITLE
Can't Import Mask from ECMA

### ISSUE THREAD

**Issue by barjabeen12** [elrumordelaluz/reactour#512]
Can't import the named export 'Mask' from non EcmaScript module (only default export is available)

**Comment by barjabeen12** [comment]
getting this error while setting up the libaray, any idea how can i solve this?

**Comment by elrumordelaluz** [comment]
Hi @barjabeen12, thanks for open the _Issue_.

Can you share a peace of code and the literal errors you are getting. Ideally, mind creating a minimal reproduction in a sandbox in order to allow others to debug your use-case and try to find a solution? Thanks!

**Comment by elrumordelaluz** [comment]
I am sorry, but as all the SO posts and related point is needed to upgrade into `ˆ5`. Any input super welcome however.

**Comment by barjabeen12** [comment]
An update
This issue was coming specifically while import tour from `@reactour/tour`. i just imported `import Tour from 'reactour'` working fine now.

**Comment by elrumordelaluz** [comment]
What version of `reactour` are you using?

**Comment by barjabeen12** [comment]
i am using `1.18.7` rn.

**Comment by elrumordelaluz** [comment]
did you switched version?

I don't understand [this](https://github.com/elrumordelaluz/reactour/issues/512#issue-1399553085) in case you were using `v1` before:

> import { TourProvider } from "@reactour/tour"

**Comment by michelonsouza** [comment]
I'm having the same problem, I just import the TourProvider and this error happens...
`react-scripts: 4.0.1`

**Comment by elrumordelaluz** [comment]
> I'm having the same problem, I just import the TourProvider and this error happens... `react-scripts: 4.0.1`

Which version of `reactour`?

**Comment by michelonsouza** [comment]
I'm using `latest` version

**Comment by elrumordelaluz** [comment]
Can't you upgrade `react-scripts` into `^5`?

**Comment by michelonsouza** [comment]
Unfortunately not... same problem with dependencies... 😭

**Comment by michelonsouza** [comment]
In this case, what do you advise me to do?

**Comment by elrumordelaluz** [comment]
I created [this sandbox](https://codesandbox.io/s/sandpack-project-forked-skooq6) which is using `@reactour/tour@3.1.6` and `react-scripts@4.0.1` to investigate, but everything seems to work as expected.

Can you fork and modify this sandbox in order to mirror what you are doing? Thanks!

**Comment by hglowiak2022** [comment]
I'm having the same issue, react-scripts at 4.0.3, "@reactour/tour" at ^3.1.6

**Comment by barjabeen12** [comment]
Hey everyone! So basically what i did was avoided using provider from `reactour/tour` and imported the Tour directly from reactour. Idk how it worked but yeah its working for me.

**Comment by hglowiak2022** [comment]
> Hey everyone! So basically what i did was avoided using provider from `reactour/tour` and imported the Tour directly from reactour. Idk how it worked but yeah its working for me.

are you able to use the hook to control it?

**Comment by barjabeen12** [comment]
Yes, i can. Further i will share a sandbox link in a while with implementation.

**Comment by elrumordelaluz** [comment]
> I'm having the same issue, react-scripts at 4.0.3, "@reactour/tour" at ^3.1.6 - I did it using simple implementation as you did.

mind sharing a _codesandbox_? thanks

**Comment by barjabeen12** [comment]
Hey, 
here is the simple implementation how i did it.
[codesandbox](https://codesandbox.io/s/reverent-meadow-5gnyq0)

**Comment by elrumordelaluz** [comment]
Thank you @barjabeen12, however seems that you are using the `v1` or reactour, while @hglowiak2022 seems to have this issue using the lasts versions of `@reactour/tour`.

![Screenshot 2022-10-12 at 08 59 55](https://user-images.githubusercontent.com/784056/195272472-a97362b6-ac8a-4daf-9993-bb94da53f54d.png)

Keep in mind that v1 doesn't make use of the Provider, and [the docs are in a different location](https://github.com/notifications#issuecomment-1273346424).

**Comment by barjabeen12** [comment]
yes that's what i was saying actually, i avoided using the latest because of this same issue.

**Comment by fauzanrh** [comment]
for someone using react-scripts 4.

using webpack configuratior like Craco:

```
webpack: {
    configure: (webpackConfig) => {
      webpackConfig.module.rules.push({
        test: /\.mjs$/,
        include: /node_modules/,
        type: "javascript/auto"
      });
      return webpackConfig;
    }
```

**Comment by fauzanrh** [comment]
> hi @elrumordelaluz @fauzanrh @barjabeen12 @michelonsouza @hglowiak2022
> 
> It's my first time using react tour, and I am getting the following error node_modules/@reactour/tour/dist/index.mjs Can't import the named export 'Mask' from non EcmaScript module (only default export is available)
> 
> ![](https://user-images.githubusercontent.com/56161353/194323351-e072bfa3-44f2-45a8-9717-28d3a2a4e061.png)
> 
> I am using, "react-scripts": "4.0.3", "react-router": "^5.2.0", "react-router-dom": "^5.2.0", "react": "^17.0.2" and TailwindCSS to style my application, after changing my postcss file as was advised in some links I was looking for help. my application lost all the styles.
> 
> Can you please assist me


"Hello Joel, for react-scripts version 4.0.3, which is the same version I am currently using, you will need to make some adjustments to the Webpack configuration within Create React App (CRA). You can find more information on this in the following link: https://github.com/elrumordelaluz/reactour/issues/512#issuecomment-1314836667. Keep in mind that while CRA utilizes Webpack, configuring it without ejecting react-scripts can be difficult. You may want to consider using a tool like CRACO or react-app-rewired to help with this process."

**Comment by fauzanrh** [comment]
sure, in that case your config would be:

```jsx
module.exports = {
  webpack: {
    configure: (webpackConfig) => {
      webpackConfig.module.rules.push({
        test: /\.mjs$/,
        include: /node_modules/,
        type: 'javascript/auto',
      });
      return webpackConfig;
    },
  },
  style: {
    postcss: {
      plugins: [
        require('tailwindcss'),
        require('autoprefixer'),
      ],
    },
  },
};
```
